washingtonpost/wpds-ui-kit
# feat: switch to font display swap

## What I did

We think `font-display: swap` is better for LCP and our use cases like rendering text ASAP.

- [ ] Test on alphas on spectrum
### Testing steps
1. Run lighthouse on https://wpds-ui-kit-git-swap.preview.now.washingtonpost.com/font-faces. see if any font related issues come back.
